tiatoolbox/annotation/storage.py

```diff
@@ -2773,17 +2868,20 @@ def pquery(
         return result
 
     def __len__(self) -> int:
+        """Return the length of the instance attributes."""
```

Comment:
More descriptive? 'Return number of annotations in the store.'

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/991322d859c0c22e7c7a1d713362944aa4317198